Thunder/template/req.html

```
@@ -1,164 +1,102 @@
+        </div>
+      </div>
+      
+      <div class="alert alert-warning" style="background-color: {{ jisshu_colours.description.background }}; color: {{ jisshu_colours.description.color }};">
+        <h6 class="fw-bold"><i class="fa-solid fa-triangle-exclamation me-2"></i> Disclaimer</h6>
+        <p class="mb-0">{{jisshu_disclaimer}}</p>
+        <div class="button-container">
+          <a href="{{jisshu_report_link}}" class="report">Report</a>
+        </div>
+      </div>
+  </div>
+  </main>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </main> ]

(tag-pair)